src/components/BurgerMenu/BurgerMenu.tsx

```diff
@@ -14,6 +14,7 @@ import { BurguerMenuItem } from './components';
 import { Separator } from '../ui/separator';
 import { SessionContext } from '@/contexts';
 import { usePartners } from '@/hooks';
+import { ToggleTheme } from '../Theme/toggle';
```

Comment:
Renomearia esse componente para algo mais explícito e não deixar dúvidas de que é um botão.
```suggestion
import { ToggleThemeButton } from '../Theme/toggle';
```